fetch.bs

```diff
@@ -4283,32 +4307,50 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
       <!-- cache hit -->
       <ol>
-       <li><p>If <var>storedResponse</var> requires validation (i.e., it is not fresh), then set the
-       <var>revalidatingFlag</var>.
-
-       <li>
-        <p>If the <var>revalidatingFlag</var> is set and <var>httpRequest</var>'s
-        <a for=request>cache mode</a> is neither "<code>force-cache</code>" nor
-        "<code>only-if-cached</code>", then:
+       <li><p>If <a for=request>cache mode</a> is "<code>default</code>" and
+        <var>storedResponse</var> is a <a>stale-while-revalidate response</a>, then:
+        <ol>
+         <li><p>Set <var>response</var> to <var>storedResponse</var>.
+         <li><p>Set <var>response</var>'s <a for=response>cache state</a> to "<code>local</code>".
+         <li><p>In parallel perform <a for=main>main fetch</a> using <var>request</var> with
+          <ul class=brief>
+          <li><p><a for=request>cache mode</a> set to "<code>no-cache</code>".
+          <li><p><a for=request>prevent no-cache cache-control header modification flag</a> set.
+          <li><p><a for=request>service-workers mode</a> set to "<code>none</code>".
+         </ul>
+
+         <p class=note>Note the parallel fetch is only meant to update the state of the HTTP cache
+         and the response will be unused until another cache access. The stale response will be used
+         as the response to current request. The parallel fetch will be issued in the context of
+         the browsing context so if it is destroyed the request will be aborted.
 
+        </ol>
+       <li><p>Otherwise:
         <ol>
-         <li><p>If <var>storedResponse</var>'s <a for=response>header list</a>
-         <a for="header list">contains</a> `<code>ETag</code>`, then <a for="header list">append</a>
-         `<code>If-None-Match</code>` with its value to <var>httpRequest</var>'s
-         <a for=request>header list</a>.
-
-         <li><p>If <var>storedResponse</var>'s <a for=response>header list</a>
-         <a for="header list">contains</a> `<code>Last-Modified</code>`, then
-         <a for="header list">append</a> `<code>If-Modified-Since</code>` with its value to
-         <var>httpRequest</var>'s <a for=request>header list</a>.
+         <li><p>If <var>storedResponse</var> is a <a>stale response</a>, then set the
+          <var>revalidatingFlag</var>.
+          <p>If the <var>revalidatingFlag</var> is set and <a for=request>cache mode</a>
+          is not one of "<code>force-cache</code>", "<code>only-if-cached</code>", then:
```

Comment:
This sentence doesn't make sense? Now that there's no new enum value the original wording should be restored I think.

Reply:
done